app/workers/comments/create_first_reaction_worker.rb

```diff
@@ -0,0 +1,18 @@
+module Comments
+  class CreateFirstReactionWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(comment_id, user_id)
+      return unless Comment.where(id: comment_id).exists?
+
+      Reaction.create(
+        user_id: user_id,
+        reactable_id: comment_id,
+        reactable_type: "Comment",
+        category: "like",
+      )
+    end
```

Comment:
Wondering why you changed the job's signature :)

https://github.com/thepracticaldev/dev.to/blob/ee8fd7cbc22f2edaced8163b4eabb192ebde5849/app/jobs/comments/create_first_reaction_job.rb#L5-L7

Reply:
I changed it to avoid loading the whole comment object. I realized that the user_id was the only other thing that was used on this job and it doesn't change after the comment is created. So I think it was a good approach to avoid loading the comment object. 

What do you think?

Reply:
I think it's a good idea :) A `SELECT EXISTS` is lighter.

I'm going to request an additional review in case I'm missing something!